packaging/Makefile.am

```diff
@@ -1,18 +1,3 @@
 EXTRA_DIST = \
-debian/README \
-debian/changelog \
-debian/charliecloud.examples \
-debian/charliecloud.install \
-debian/compat \
-debian/control \
-debian/copyright \
-debian/docs \
-debian/format \
-debian/rules \
-debian/travis.sh \
-debian/watch \
-fedora/build \
```

Comment:
We don't want to remove the fedora packaging bits do we?

Reply:
![foo](https://media.giphy.com/media/3o7btUg31OCi0NXdkY/giphy.gif)